src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFHeaderLines.java

```diff
@@ -62,6 +62,10 @@ private static void addFilterLine(final VCFFilterHeaderLine line) {
         addFilterLine(new VCFFilterHeaderLine(READ_ORIENTATION_ARTIFACT_FILTER_NAME, "orientation bias detected by the orientation bias mixture model"));
         addFilterLine(new VCFFilterHeaderLine(BAD_HAPLOTYPE_FILTER_NAME, "Variant near filtered variant on same haplotype."));
 
+        //Mitochondrial M2-related filters
+        addFilterLine(new VCFFilterHeaderLine(CHIMERIC_ORIGINAL_ALIGNMENT_FILTER_NAME, "NuMT variant with too many ALT reads originally from autosome"));
+        addFilterLine(new VCFFilterHeaderLine(LOW_AVG_ALT_QUALITY_FILTER_NAME, "Low average alt quality"));
+        addFilterLine(new VCFFilterHeaderLine(LOW_LOD_FILTER_NAME, "Variant does not meet likelihood threshold"));
```

Comment:
We could unify some of the mito and tumor filter and info fields by making the M2 names less tumor specific.  If Lee and Geraldine are okay with it I have no problem with eg TLOD -> LOD.

Reply:
That would be helpful. Is that difficult for legacy reasons? (Will users complain about the change breaking their downstream scripts?)